tbx/static_src/sass/components/_featured-services.scss

```diff
@@ -0,0 +1,48 @@
+@use 'config' as *;
+
+.featured-services {
+    &__intro {
+        margin-bottom: $spacer-medium;
+    }
+
+    &__cards {
+        display: grid;
+        gap: $spacer-mini-plus;
+        grid-template-columns: repeat(1, 1fr);
+    
+        @include media-query(medium) {
+            grid-template-columns: repeat(2, 1fr);
+        }
+    
+        @include media-query(large) {
+            grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
+        }
+    }
+    
+    &__card {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        transition: background-color $transition-quick;
+        background-color: var(--color--dark-background);
+
+        &:has(.button-link:hover) {
+            background-color: var(--color--black);
+        }
+    }
+
+    &__text {
+        padding: $spacer-small;
+        margin-bottom: auto;
```

Comment:
Praise: nice bit off CSS here to keep all the buttons at the bottom of their containers regardless of the amount of text above them : )